src/Pages/Login/Login.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { SEND_TO_SERVER } from '../../config';
+
+export default function Login() {
+  const { Kakao } = window;
+  let history = useHistory();
+
```

Comment:
```suggestion
import { SEND_TO_SERVER } from '../../config';

const { Kakao } = window;

export default function Login() {
  const history = useHistory();

```
- html head 태그에서 카카오 스크립트를 이미 부르고 있기 때문에 컴포넌트 바깥으로 빼주어도 됩니다
- 특별한 이유 없다면 const로 선언해주세요

Reply:
수정하겠습니다~